Webui page title customization

benji101 opened the pull request:
**Describe what this pull request is trying to achieve.**

Adding the possibility to customize the webui page title with a command argument.

**Additional notes and description of your changes**

I think this could be useful when people have more than one instance running to easily identify on which environnement they are connected.

The argument is --ui-webpage-title and defaults to "Stable Diffusion" like before the change.

**Environment this was tested in**

List the environment you have developed / tested this on. As per the contributing page, changes should be able to work on Windows out of the box.
 - OS: Windows
 - Browser: chrome
 - Graphics card: NVIDIA 3060

Screenshot:
![Capture d'écran 2023-05-07 185649](https://user-images.githubusercontent.com/1480586/236706658-b04ed99b-d6c3-460a-b52d-03cce950b48a.png)

benji101 committed:
Update cmd_args.py

benji101 committed:
Update ui.py

missionfloyd commented:
It needs to go here too.
https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/5ab7f213bec2f816f9c5644becb32eb72c8ffb89/javascript/progressbar.js#L49-L59

benji101 committed:
Moved the option from command arguments to ui settings

Added missing title update to progressbar.js
Moved the title setting from command argument to ui web setting

benji101 commented:
I think it would be better to add this setting directly in the webui settings page. Here's a screenshot of the result:

![screenshot 2023-05-11 101847](https://github.com/AUTOMATIC1111/stable-diffusion-webui/assets/1480586/83575c92-d4f2-4577-94a7-347253f4fab9)

I also added the missing progressbar title update.

AUTOMATIC1111 commented:
This is a small and simple addition but due to how little use this feature would i imagine see I'd prefer if it was done as an extension.
